examples/aws/terraform/ha-autoscale-cluster/proxy-user-data.tpl

```diff
@@ -14,3 +14,8 @@ TELEPORT_ENABLE_POSTGRES=${enable_postgres_listener}
 USE_ACM=${use_acm}
 USE_TLS_ROUTING=${use_tls_routing}
 EOF
+cat >>/etc/default/teleport <<EOF
```

Comment:
Is /etc/teleport.d/conf also necessary?

Reply:
Yeah, that file is sourced all over the place.

I could theoretically rewrite all the places it's used to use `/etc/default/teleport` instead, but setting a lot of extra environment variables in the environment file Teleport itself sources might have unintended consequences. It'd also be a huge PR that would need a lot of testing.

At the moment there's a relatively clean split between "environment variables used by Teleport" and "environment variables used by the AMI/unit files".